jakarta.ls/src/main/resources/persistence.json

```diff
@@ -51,7 +51,7 @@
 	      "@Entity",
 	      "@Table(name = \"${2:entityName}\")",
 	      "@NamedQuery(name=\"\", query=\"\")",
-	      "public class ${2:classname} {",
```

Comment:
This fixes the class name but the same thing needs to be done to the constructor.

Reply:
Thanks for catching this, will fix